### DISCUSSION TITLE
How to remove the ability to download the entire folder ?

### DISCUSSION THREAD

**Discussion by Kineshma** [drakkan/sftpgo#1009] [Q&A] [answered]
![2022-10-04_10-58-22](https://user-images.githubusercontent.com/15850287/193766658-59eef888-ab4e-4a6f-9987-bb1aaa8d0483.png)

How to remove the ability to download the entire folder ? 
only single files, not the entire folder

**Comment by drakkan** [comment, marked as answer]
Hi,

give a try to the development version, you can select the files to download as in the WebClient UI
